quill-core/src/main/scala/io/getquill/dsl/MetaDslMacro.scala

```diff
@@ -58,7 +58,7 @@ class MetaDslMacro(val c: MacroContext) extends ValueComputation {
       q"""
           new ${c.prefix}.SchemaMeta[$t] {
             private[this] val _entity =
-              ${c.prefix}.quote(${c.prefix}.querySchema[$t](${t.tpe.typeSymbol.name.decodedName.toString}))
+              ${c.prefix}.quote(${c.prefix}.impliedQuerySchema[$t](${t.tpe.typeSymbol.name.decodedName.toString}))
```

Comment:
Need to have a separatae `impliedQuerySchema` method so that the parser knows to not to set the resulting `Entity` class's `renamable` field to `Fixed`. Basically, now all `querySchema` elements will have their `Entity.renameable` set to `Fixed` and `impliedQuerySchema` elements will have their `Entity.renameable` set to `ByStrategy`.